### PR TITLE
Allow remaster in Naming/InclusiveLanguage

### PR DESCRIPTION
**Pull request by kpost**
In the same spirit as 'Mastercard' and 'webmaster', 'remaster' (as in audiophonic or videographic remastering of older recordings etc) should be allowed as well.

### CODE REVIEW

**Comment by larouxn** [comment]
👋 @kpost, looks like you still need to [sign the CLA](https://cla.shopify.com)https://cla.shopify.com/. Can you do so when you get a chance? ([logs](https://github.com/Shopify/ruby-style-guide/actions/runs/5806301972/job/15738919622?pr=564))

**Comment by kpost** [comment]
@larouxn Done. However it seems I can't rerun the job.

File: rubocop.yml [hunk 1]
```diff
@@ -383,6 +383,7 @@ Naming/InclusiveLanguage:
         - !ruby/regexp '/origin[ \/]master/' # Legacy default git branch name
         - 'mastercard'
         - 'webmaster'
+        - 'remaster'
```
Comment (sambostock) [inline rubocop.yml:386]: Should we just allow any unbroken word that contains "master"? Something like `/\wmaster\w/`.

[the remark this answers is not in the document]
Comment (kpost) [reply rubocop.yml:386]: I agree, but I figured lots of words that include 'master' could still be considered not inclusive. Like 'mastering' or something? But I'm not really sure what is considered bad or good in this context anymore tbh.

[the remark this answers is not in the document]
Comment (kpost) [reply rubocop.yml]: Definitely. Done. Apologies, didn't get around to it last couple days.

So I noticed there aren't any unit tests or something in place to test these changes in the default Rubocop config? Is that correct?

**Review by sambostock — APPROVED** [review]

**Comment by sambostock** [comment]
Thank you @kpost!

File: rubocop.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (kpost) [reply rubocop.yml]: Right. It feels like it wouldn't be a bad idea though. At this point writing rubocop config almost feels like a DSL and testing the config more and more necessary. For now its fine though. Thanks for merging!

**Pull request merged**